app/views/podcasts/index.html.erb

```diff
@@ -1,35 +1,61 @@
-<div class="jumbotron-jumbotron-fluid">
-  <h1 class="display-4">Hello, world!</h1>
-  <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+<div class="jumbotron-jumbotron-fluid center-search">
+  <h1 class="display-4">Podsmack</h1>
+  <p class="lead">Your next favorite podcast is waiting to be found!</p>
   <hr class="my-4">
-  <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
-  <a class="btn btn-primary btn-lg" href="#" role="button">Learn more</a>
+  <p>Select exactly what you're looking for.</p>
 </div>
 <br>
 <br>
+<br>
+
+
+
 <%= form_tag(podcasts_path, method: "get") do %>
-  <%= label_tag 'Location' %>
-  <%= select_tag(:location, options_for_select([nil, 'Denver', 'NYC', 'Dallas', 'San Francisco']), { class: "btn btn-outline btn-xl js-scroll-trigger" })%>
+  <div class="container">
+    <div class="row">
 
-  <%= label_tag :tags %>
-  <%= collection_check_boxes(:tags, :ids, Tag.all, :id, :name, include_hidden: false) %> <br>
+        <div class="col-sm center-search">
+          <%= label_tag 'Location' %>
+          <%= select_tag(:location, options_for_select([nil, 'Denver', 'NYC', 'Dallas', 'San Francisco']), { class: "btn btn-outline btn-xl js-scroll-trigger" })%>
+        </div>
 
-  <%= label_tag 'Exclude Adult Content' %>
-  <%= check_box_tag :adult_content, checked = false %> <br>
+        <div class="col-sm center-search">
+          <%= label_tag :tags %>
+          <%= collection_check_boxes(:tags, :ids, Tag.all, :id, :name, include_hidden: false) do |box| %><br>
+            <%= box.label {box.check_box + box.text} %> 
+          <% end %>
+        </div>
 
-  <%= label_tag :name %>
-  <%= text_field_tag :name %> <br>
+        <div class="col-sm center-search">
+          <%= label_tag 'Exclude Adult Content' %>
+          <%= check_box_tag :adult_content, checked = false %> 
+        </div>
 
+        <div class="col-sm center-search">
+          <%= label_tag :name %>
+          <%= text_field_tag :name %> 
+        </div>
 
-  <%= submit_tag 'Search' %>
+    </div>
+  </div>
+  <br>
+  <br>
+  <div class='center-search'>
+    <%= submit_tag 'Search' %>
+  </div>  
 <% end %>
-
-<h3> Podcast Collection </h3>
-<section class='podcast-browse'>
-  <h1><%= @podcasts.length %> Results </h1>
-  <% @podcasts.each do |podcast| %>
-    <section id='podcast_results'>
-      <%= podcast.name %>, <%= podcast.description %>
-    </section>
-  <% end %>
-</section>
\ No newline at end of file
+<br>
+<br>
+  <h3 class="center-search"> Podcast Collection </h3>
+<br>
+<br>
+  <section class='podcast-browse center-search'>
+    <h1><%= @podcasts.length %> Results </h1>
+    <% @podcasts.each do |podcast| %>
+      <section id='podcast_results'>
+        <%= link_to podcast.name, podcast_path(podcast.id) %>
```

Comment:
Each podcast name is now a link to their show page. However, we ran into an error but I think it had to do with our test data not ACTUALLY being a podcast on the site ie approved.